src/utils/semanticUtils.js

```diff
@@ -22,6 +22,7 @@ export const floats = ['left', 'right']
 export const iconCategories = [
```

Comment:
In light of this comment https://github.com/TechnologyAdvice/stardust/pull/279/commits/dd443b5c0d0c5b6d585ee0386bf438cfee8da854#r70285412, I think this util is becoming more of a doc site util.  Maybe we should consider separating the doc utility from the library utility.

I see the library needs an array of icon names.  The docs need those same icon names, but with category headings and descriptions.  Here's one thought:

```js
// src/utils/semanticUtils.js

export const webContentIcons = [
  'add to calendar', 'alarm outline', ...
]
export const userActions = [
  'add to cart', 'add user', 'adjust', ...
]
export const icons = [
  ...webContentIcons,
  ...userActions,
  // etc.
]
```

```js
// docs/app/utils/icons.js

import { webContentIcons, userActions } from 'src/utils/semanticUtils'

export const iconsByCategory = [
  {
    name: 'Web Content',
    description: 'Icons can represent types of content found on websites',
    icons: webContentIcons,
  },
  {
    name: 'User Actions',
    description: 'Icons can represent common actions a user can take',
    icons: userActions,
  },
]
```